src/partner/modules/EventList/EventListItem.tsx

```diff
@@ -1,11 +1,14 @@
 import React from "react";
 import { Link } from "react-router-dom";
-import { ListStyled } from "src/partner/modules/ui";
 import { IEvent } from "src/partner/models/Event";
 import { Modal } from "src/partner/modules/ui/Modal/Modal";
 import { Notification } from "src/partner/modules/ui/Modal/Notification";
 import { CreateEvent } from "src/components/event/Create";
 import styledComponents from "styled-components";
+import {
+  ListStyled,
+  NotificationContextProvider
```

Comment:
`NotificationContextProvider` should be in `providers/NotificationContextProvider`